OpenQA/Isotovideo/CommandHandler.pm

```diff
@@ -350,4 +351,37 @@ sub update_status_file ($self) {
     rename $tmp, AUTOINST_STATUSFILE or die $!;
 }
 
+my ($last_check_seconds, $last_check_microseconds) = gettimeofday;
```

Comment:
These variables could now be fields of the object. Otherwise it'll be a bit unintuitive that different instances influence each other this way.